packages/create-turbo/src/commands/create/updateCommandsInREADME.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs").promises;
+
+const filePath = "../../../../../examples/basic/README.md";
+
+async function updateCommandsInREADME(selectedPackageManager) {
+  try {
+    // Read the content of the file
+    let data = await fs.readFile(filePath, "utf8");
+
+    // Replace all occurrences of 'pnpm' with selectedPackageManager
+    data = data.replace(/\bpnpm\b/g, `${selectedPackageManager} run`);
```

Comment:
This should work for all package managers, so we should be able to transform from `{npm|pnpm|yarn|bun}` to `{npm|pnpm|yarn|bun}`

Reply:
Done